tfx/experimental/distributed_inference/graphdef_experiments/subgraph_partitioning/create_complex_graph.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Generate an example consists of {'main', 'remote_op_a', 'remote_op_b'}
+
+@author: jzhunybj
+"""
+
+
+import tensorflow as tf
+import numpy as np
+
+
+tf.compat.v1.disable_eager_execution()  # Disable eager mode
+
+N = 1000    # number of embeddings
+NDIMS = 16  # dimensionality of embeddings
+
+
+def create_session(graph):
+    return tf.compat.v1.Session(graph=graph,
+                                config=tf.compat.v1.ConfigProto(
+                                        inter_op_parallelism_threads=8))
+
+
+tf.random.set_seed(5)  # Deterministic results
```

Comment:
I believe this should be placed right before sess.run, but I'm not sure. @goutham what's your suggestion?

Reply:
Removed since it wasn't effectively being used